week-3/extended-project/nowhere/views/index.erb

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>
+      Nowhere
+    </title>
+    <!-- Meta tag Keywords -->
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta charset="UTF-8" />
+    <meta name="keywords" content="Hi Codeable!" />
+    <script>
+      addEventListener(
+        'load',
+        function() {
+          setTimeout(hideURLbar, 0);
+        },
+        false
+      );
+
+      function hideURLbar() {
+        window.scrollTo(0, 1);
+      }
+    </script>

```

Comment:
I recommend you that you don't add the tag "script" - to write javascript code or reference to javascript files - at the above of a page because the browser will not show HTML or anything to the user while the browser don't finish in reading and validating the file javascript and generating a bad experience to the user.